### DISCUSSION TITLE
Is it possible to pull tables from the database?

### DISCUSSION THREAD

**Discussion by geenva** [prisma/prisma#9070] [Q&A] [answered]
Pretty much `prisma pull dev`, instead of push. Is it possible for us to get existing tables from the database?

**Comment by janpio** [comment, marked as answer]
Of course, the feature you are looking for is called Introspection and uses the command `prisma db pull`: https://www.prisma.io/docs/concepts/components/introspection

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
